utils/components-utils/src/main/scala/pl/touk/nussknacker/engine/util/watermarkstrategy/WatermarkStrategyValidationHandler.scala

```diff
@@ -57,7 +57,7 @@ trait WatermarkStrategyValidationHandler { self: SingleInputDynamicComponent[_]
           outputValidationContext.localVariables.mapValuesNow(AdditionalVariableProvidedInRuntime(_)),
         defaultValue = Some(eventTimeDefaultValueExpression),
         hintText = Some(
-          s"An expression that determines the Event Time to be used in stateful stream processing. " +
+          s"An expression that determines the Event Time to be used in stateful stream processing. If this field won't be specified, the upstream source event time will be used.  " +
```

Comment:
```suggestion
          s"An expression that determines the Event Time to be used in stateful stream processing. If this field is empty, the upstream source event time will be used.  " +
```

Reply:
I think from user perspective its better to say that field `is empty` than field `is specified`. And I think present tense in first part of conditional is correct (its first or even zero conditional)

Reply:
You are right, my mistake, Thank you for catching it out!